.circleci/config.yml

```diff
@@ -0,0 +1,28 @@
+version: 2.0
+
+jobs:
+  build:
+    docker:
+      - image: cibuilds/docker:19.03
+    steps:
+      - checkout
+      - setup_docker_php
+      - run:
+          name: "Build Docker Image"
+          command: docker build --file Dockerfile
+      - deploy:
```

Comment:
the Deploy step is not needed I think. I have docker hub integration in place that should listen to new commits in master and do that thing on its own, or?